metadata-load/src/main/resources/application.properties

```diff
@@ -30,3 +30,7 @@ spring.main.web-environment=false
 entrez.api.key=@entrez.api.key@
 entrez.api.attempts=10
 entrez.api.delay=500
+
+# ENA API settings to accommodate slow response
```

Comment:
This needs to be slightly rephrased: what we're fixing here is not really a slow response, but _rare and sporadic errors_